board/flags-config.cmake

```diff
@@ -0,0 +1,37 @@
+# Flags for each supported board
+
+function(flags BOARD_NAME)
+  # Set all minimization flags for all platforms
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Os -fdata-sections -ffunction-sections -fomit-frame-pointer -fno-unwind-tables -fno-asynchronous-unwind-tables -fvisibility=hidden -fshort-enums -finline-small-functions -findirect-inlining -fno-common") 
```

Comment:
Might be useful to break that line down to make it easier to read.

Reply:
I tried to, but CMake does not like it, is there any way to break lines in CMake?

Reply:
https://stackoverflow.com/questions/7637539/how-to-split-strings-across-multiple-lines-in-cmake

You could also do something like this:

```
set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Os")
set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fdata-sections")
...
```